src/main/java/com/kodilla/ecommercee/dto/OrderDto.java

```diff
@@ -1,14 +1,27 @@
 package com.kodilla.ecommercee.dto;
+import com.kodilla.ecommercee.GenericEntity;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
+import javax.persistence.*;
+import java.util.Date;
+import java.util.List;
+
 @Getter
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
 public class OrderDto {
 
     private Long Id;
+
+    private Date orderMade;
+
+    private Boolean resolved;
```

Comment:
`private Boolean resolved;`
Jeśli nie planujesz przechowywać tutaj nulla to można zrobić tym prymitywny.